db/patterns/crazy_egg.eno

```diff
@@ -6,6 +6,7 @@ organization: crazy_egg
 --- domains
 cetrk.com
 crazyegg.com
+script.crazyegg.com
```

Comment:
trackerdb and some Ghostery applications like Anti-Tracking and WhoTracks.Me operate on a level of eTLD+1. Subdomains like this will simply be ignored. Maybe we can add a unit test to test for this.

Reply:
I dropped non mandatory subdomains not required for explicit detection.